examples/web_apps/webxr_hand_demo/js/vr_demo.js

```diff
@@ -65,39 +61,67 @@ export class VRDemo {
   }
 
   static preloadFiles(preloadFunc) {
-    preloadFunc(DataUtils.getPhysicsConfigFilepath());
+    function doPreload(file) {
+      preloadedFiles.push("../" + file);
+      preloadFunc(file);
+    }
+
+    doPreload(DataUtils.getPhysicsConfigFilepath());
 
-    preloadFunc(DataUtils.getStageFilepath(Module.stageName));
-    preloadFunc(DataUtils.getStageConfigFilepath(Module.stageName));
+    doPreload(DataUtils.getStageFilepath(Module.stageName));
+    doPreload(DataUtils.getStageConfigFilepath(Module.stageName));
 
-    preloadFunc(DataUtils.getObjectFilepath("hand_r_open"));
-    preloadFunc(DataUtils.getObjectConfigFilepath("hand_r_open"));
-    preloadFunc(DataUtils.getObjectFilepath("hand_r_closed"));
-    preloadFunc(DataUtils.getObjectConfigFilepath("hand_r_closed"));
+    doPreload(DataUtils.getObjectFilepath("hand_r_open"));
+    doPreload(DataUtils.getObjectConfigFilepath("hand_r_open"));
+    doPreload(DataUtils.getObjectFilepath("hand_r_closed"));
+    doPreload(DataUtils.getObjectConfigFilepath("hand_r_closed"));
 
-    preloadFunc(DataUtils.getObjectFilepath("hand_l_open"));
-    preloadFunc(DataUtils.getObjectConfigFilepath("hand_l_open"));
-    preloadFunc(DataUtils.getObjectFilepath("hand_l_closed"));
-    preloadFunc(DataUtils.getObjectConfigFilepath("hand_l_closed"));
+    doPreload(DataUtils.getObjectFilepath("hand_l_open"));
+    doPreload(DataUtils.getObjectConfigFilepath("hand_l_open"));
+    doPreload(DataUtils.getObjectFilepath("hand_l_closed"));
+    doPreload(DataUtils.getObjectConfigFilepath("hand_l_closed"));
 
     const replicaCadObjectNames = new Set();
     for (const object of objectSpawnOrder) {
       replicaCadObjectNames.add(object);
     }
 
     for (const name of replicaCadObjectNames) {
-      preloadFunc(DataUtils.getObjectFilepath(name));
-      preloadFunc(DataUtils.getObjectCollisionGlbFilepath(name));
-      preloadFunc(DataUtils.getObjectConfigFilepath(name));
+      doPreload(DataUtils.getObjectFilepath(name));
+      doPreload(DataUtils.getObjectCollisionGlbFilepath(name));
+      doPreload(DataUtils.getObjectConfigFilepath(name));
     }
   }
 
+  applyKeyframe(jsonKeyframe) {
+    let keyframe = this.player.keyframeFromString(jsonKeyframe);
```

Comment:
How about a version of `applyKeyframe` that takes a JSON keyframe? Then we never have to construct a Javascript Keyframe object, right? Maybe it'll faster.

With that said, I don't want you to entirely remove the Keyframe bindings and related functionality you've shown here. I can imagine a future use where the app wants to manipulate the keyframe in Javascript.